config/webpack.common.js

```diff
@@ -2,17 +2,17 @@
  * @author: @AngularClass
  */
 
-var webpack = require('webpack');
-var helpers = require('./helpers');
+const webpack = require('webpack');
+const helpers = require('./helpers');
```

Comment:
@gdi2290, I thought because we have

```JSON
"engines": {
    "node": ">= 4.2.1",
    "npm": ">= 3"
}
```

we can assume that the user will have node 4.2.1+, no? Unfortunately though, you are right. There won't be any warning messages. However, we are already using `const` elsewhere (e.g. for the environment variables in the Webpack config files) so the app would break anyways. That's why I would vote for keeping `const` here.